NppMenuSearch/NppPluginNETHelper.cs

```diff
@@ -6,422 +6,422 @@
 

```

Comment:
Seems there are plenty of whitespace differences which might be not necessary and it would make file review easier if just the relevant changes appear

Reply:
Oh? On my site Visual Studio is showing the actual changes only:

![grafik](https://user-images.githubusercontent.com/4410841/51496549-81326280-1dc0-11e9-8396-a74e20cea4bc.png)

Reply:
Dear @budul100, 
thank you very much for your contribution. To help reviewing the code changes here on github, could you please use an indentation of 4 spaces instead of tabs, as in the rest of the code? A change of the indentation style should be a different pull request if it is really necessary.
Thank you,
Peter

Reply:
Dear Peter, No Problem. I am quite new in posting on GitHub. I've changed the file accordingly and commited this change (see 073d85b57a96504dba4c0df3985dedb603f09bae).